### DISCUSSION TITLE
RSSI Setting

### DISCUSSION THREAD

**Discussion by dsandels** [cyberman54/ESP32-Paxcounter#1019] [General]
Hello all,

Has anyone ever tested the Set RSSI feature to determine the actual reception radius for different values of RSSI ?

Thanks,

Dave

**Comment by cyberman54** [comment]
Sure, but it depends on your antenna.

**Comment by dsandels** [comment]
Thanks for your reply !

How about with the LilyGo onboard antenna ?

Dave

On Mon, Aug 12, 2024 at 5:20 PM Verkehrsrot ***@***.***>
wrote:

> Sure, but it depends on your antenna.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cyberman54/ESP32-Paxcounter/discussions/1019#discussioncomment-10319266>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APPZGZIN2JYAUYA2R4SMW33ZREYMRAVCNFSM6AAAAABMNAQPOOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZRHEZDMNQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <cyberman54/ESP32-Paxcounter/repo-discussions/1019/comments/10319266@
> github.com>
>


-- 
David Sandel
iPhone 314-435-3658
Fax 314-862-4937
Twitter @dsandel @iNeighborhoods @EDesignsCenter
Founder i*iNeighborhoods <http://ineighborhoods.us/>*
6900 Delmar, St. Louis, Missouri 63130
MapSTL <http://ineighborhoods.us/digitaldivide> - Mapping the Digital Divide
My LinkedIn Profile <http://linkedin.com/in/smartcitymasterplanner>
